modules/client-configuration/pages/activation-keys.adoc

```diff
@@ -11,12 +11,12 @@ When the bootstrap script is executed all configuration settings associated with
 
 An activation key can specify:
 
-* Channel Assignment
-* System Types (Traditionally called Add-on Entitlements)
-* Contact Method
-* Configuration Files
-* Packages to be Installed
-* System Group Assignment
+* Channel assignment
+* System types (traditionally called add-on entitlements)
```

Comment:
Do you think that using "traditionally" here might be confusing? Perhaps we could do something like this instead:

```suggestion
* System types (or add-on entitlements)
```
```suggestion
* System types (traditionally called add-on entitlements)
```

Reply:
yes, your proposal is better.